app/controllers/favorites_controller.rb

```diff
@@ -0,0 +1,46 @@
+require 'TunesTakeoutWrapper'
+
+
+class FavoritesController < ApplicationController
+
+  def create
+    status_code = TunesTakeoutWrapper.add_favorite(session[:user_id], params[:suggestion_id])
+
+    if status_code == 201
+      @message = "Favorite sucessfully added!"
+    elsif status_code == 409
+      @message = "You've already favorited this pairing."
+    else
+      @message = "Sorry, we were unable to save your favorite at this time."
+    end
+
+    @status_code = status_code
```

Comment:
Is there a reason you can't assign `@status_code` to your the result of your method call on line 7?